_redirects

```diff
@@ -25,3 +25,4 @@
 /recipes/direnv /guides/recipes/direnv 301
 /tutorials/learning-journey/sharing-dependencies /guides/recipes/sharing-dependencies 301
 /tutorials/learning-journey/packaging-existing-software /tutorials/packaging-existing-software 301
+/permalink/stub-ld /guides/faq#how-to-run-non-nix-executables 301
```

Comment:
```suggestion
# incoming permalinks
/in/stub-ld /guides/faq#how-to-run-non-nix-executables 301
```

Brevity worth considering.

Reply:
I feel like that's a bit too much brevity. Having it be obvious what it's for just looking at the url is more useful than it being really short, I think.

Reply:
I think `permalink` is a little better, but I don't care much either way.

`topic` *might* conflict with a future desired organizational structure. Low chance, though.

Reply:
I'm also for `permalink`, it expresses exactly what it's for.